test/requestBody.test.js

```diff
@@ -416,6 +416,13 @@ describe('request body handling', function () {
         expect(har.log.entries[0].request.postData.text).to.equal(JSON.stringify({ a: 'test' }));
       });
 
+      it('should work for RAW_BODY objects (but data is a primitive somehow)', function () {
```

Comment:
This case should never actually happen in real life, but just in case the `RAW_BODY` property is defined as an object in the spec but the `formData` we have is a primitive we still shouldn't stringify it.